about.html

```diff
@@ -11,54 +11,59 @@
 
 </head>
 
+<header>
+  <nav>
+      <img src="/img/lambda-black.png" alt="logo">
+      <a href="/index.html">Home</a>
+      <a href="/about.html">About</a>
+      <a href="">Products</a>
+      <a href="">Blog</a>
+      <a href="">Contact</a>
+  </nav>
+  <section class = 'hero'>
+      <img src="/img/jumbo-about.png" alt="">
+  </section>
+</header>
+
 <body>
     <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
```

Comment:
Good job laying out your sections and using descriptive class names that tells where the section is laid out